apps/snitch_core/lib/core/domain/order/transitions.ex

```diff
@@ -124,4 +127,57 @@ defmodule Snitch.Domain.Order.Transitions do
   end
 
   def persist_shipment(%Context{valid?: false} = context), do: context
+
+  defp process_package(package, shipping_method_id) do
+    shipping_method =
+      Enum.find(package.shipping_methods, fn %{id: id} ->
+        id == shipping_method_id
+      end)
+
+    package_total =
```

Comment:
Alongwith the `package_total`, we must compute the `tax_total`, `promo_total` and `adjustment_total` as well. Just set them to zeroes.
When the relevant `Domain` functions are added, we'll call them here.